data/changes/bantu.txt

```diff
@@ -15,26 +15,26 @@ Bantu
 - [*p̱ *t *k] / [{p s̪} t̪] / _ [*i̧ *u̧]
 - *ʃ / s̪ / _ *i̧
 + Inferred from modern reflexes
-- [*ḇ *ḏ *g̱] / {*ḇ z̪} {*ḏ z̪} {*g̱ z̪} / _ [*i̧ *u̧]
+- [*ḇ *ḏ *g̱] / [{*ḇ *z̪} {*ḏ *z̪} {*g̱ *z̪}] / _ [*i̧ *u̧]
 + ‘no reflex discovered’ for '*ji̧' and '*ju̧'
 - [*m *n] / n̪ / _ *i̧
 + ‘no reflex discovered’ for '*mu̧' and '*nu̧'
 - [*ᵐb *ⁿd *ᵑg] / ⁿd̪ / _ *i̧
-- {p s̪} / s̪
+- p s̪ / s̪
 - t̪ / s̪ / _ *i̧
-- {*ḇ z̪} {*ḏ z̪} {*g̱ z̪} / z̪
+- [{*ḇ *z̪} {*ḏ *z̪} {*g̱ *z̪}] / z̪
 - [*i̧ *u̧] / [*ɪ u]
 - *ɪ / e
 + Merger with '*e'
 - *g̱ / y
-- [*ḇ *g̱] / w _ [*o *u]
+- [*ḇ *g̱] / w / _ [*o *u]
 + Ordering unclear
 - *dʒ / *ʒ
-- [+high +long] [-high] / [+high -long] [-high]
```

Comment:
Unclear how to express features in Brassica notation, especially syllabic-level features such as tone; the source grouped vowel length and tone together in the changes, literally:

> H-L < *Hː-L

Reply:
Ah, I didn’t realise this was at the syllabic level. I will admit that Brassica currently doesn’t have great support for syllabic features, which is a significant issue for this application. (Though I do have some ideas around supporting autosegments… perhaps I should implement that sooner rather than later.)

For now, it may be best to match consonants in-between. It looks like Otjiherero has maximal CCV syllable structure, so this should be valid:

```
- [V +high +long] (C) (C) [V +low] / [V +high -long] (C) (C) [V +low]
```

Not great, but it works.

Reply:
It is valid, yes.